src/components/ArticleDetail.jsx

```diff
@@ -14,24 +16,54 @@ function ArticleDetail() {
       .catch((error) => {
         console.error("Error fetching article:", error);
       });
+
+    fetchCommentsByArticleId(article_id)
+      .then((fetchedComments) => {
+        setComments(fetchedComments);
+        setLoadingComments(false);
+      })
+      .catch((error) => {
+        console.error("Error fetching comments:", error);
+        setLoadingComments(false);
+      });
   }, [article_id]);
 
   if (!article) return <p>Loading...</p>;
+  if (loadingComments) return <p>Loading comments...</p>;
 
   return (
-    <div className="article-detail">
+    <div className="article-detail-container">
       <header>
         <h1>Article Detail</h1>
       </header>
-      <h2>{article.title} <span>{new Date(article.created_at).toLocaleString()}</span></h2>
-      <img src={article.article_img_url} alt={article.title} />
-      <p>{article.body}</p>
-      <p>Author: {article.author}</p>
+      <div className="article-detail">
+        <h2>
+           {article.title}{" "}
+           <span>{new Date(article.created_at).toLocaleString()}</span>
+        </h2>
+        <img src={article.article_img_url} alt={article.title} />
+        <p>{article.body}</p>
+        <p>Author: {article.author}</p>
+      </div>
 
-      {/* Placeholder for comments section */}
       <div className="comments-section">
         <h3>Comments</h3>
-        <p>View and post comments here...</p>
+        <div className="comments-grid">
+          {comments.length > 0 ? (
```

Comment:
is this necessary ? Think about what will happen if there are no comments. What will the state look like if there are no comments. Use article 37 as an example it doesn't have any comments